src/index.ts

```diff
@@ -0,0 +1,65 @@
+import { Editor } from 'grapesjs'
+import { KeymapsDialogManager } from './KeymapsDialogManager'
+
+const cmdKeymapsDialog = 'open-keymaps-dialog'
+
+export default (editor: Editor, opts = {}): void => {
+  const options = {
+    ...{
+      longPressKey: 'shift',
```

Comment:
did you see the long press for help pannel somewhere? is it in a software you use?